apps/studio/src/pages/sites/[siteId]/index.tsx

```diff
@@ -21,14 +31,23 @@ const SitePage: NextPageWithLayout = () => {
           </Text>
           <HStack w="100%" justify="space-between" align="center" gap={0}>
             <Box />
-            <HStack gap="0.25rem" justifySelf="flex-end">
-              <Button variant="outline" size="xs">
-                Create a folder
-              </Button>
-              <Button onClick={onPageCreateModalOpen} size="xs">
-                Create a new page
-              </Button>
-            </HStack>
+            <Menu isLazy size="sm">
+              <MenuButton as={Button} size="xs" justifySelf="flex-end">
+                Create new...
+              </MenuButton>
+              <MenuList>
```

Comment:
i don't want to rebase anymore ><, but I've added a portal to the menulist (following [chakra docs](https://v2.chakra-ui.com/docs/components/menu#rendering-menu-in-a-portal)) so they are not constrained inside the table

Reply:
nvm i still did it